### DISCUSSION TITLE
Conversion of faster_rcnn_r50_fpn_2x_coco model to onnx

### DISCUSSION THREAD

**Discussion by nyanmn** [open-mmlab/mmdetection#8789] [Q&A] [answered]
I like to convert faster_rcnn_r50_fpn_2x_coco pytorch model to onnx. It is not listed in the "List of supported models exportable to ONNX" on this [page](https://mmdetection.readthedocs.io/en/stable/tutorials/pytorch2onnx.html).
Is this faster_rcnn_r50_fpn_2x_coco pytorch model convertible to onnx?

**Comment by ZwwWayne** [comment, marked as answer]
Yes. The schedule of 1x and 2x only changes the training setting but does not change the model architecture.
